keystone-shell/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.6-alpine3.6
+
+# To force a rebuild, pass --build-arg REBUILD="$(DATE)" when running
+# `docker build`
+ARG REBUILD=1
+
+COPY requirements.txt stack-fix.c /
+
+RUN apk add --no-cache ca-certificates tini libffi && \
+    apk add --no-cache --virtual build-dep \
+        musl-dev linux-headers git make g++ libffi-dev openssl-dev && \
+    gcc -shared -fPIC /stack-fix.c -o /stack-fix.so && \
+    pip install urllib3 ipaddress ipython ptpython python-openstackclient && \
```

Comment:
With monasca/python we could always install latest master, but I am not really sure if that's needed at all ?

Reply:
Yeah, you're right, this should really be using monasca/python. The docker stuff is mostly a copy&paste hack job that came out of my frustration while debugging keystone-init.

I'm fine with holding off on merging until it's converted, though.